src/commands/BeautifyCommand.ts

```diff
@@ -84,9 +80,9 @@ export class BeautifyCommand extends BaseCommand {
       );
   }
 
-  protected get isTerminal(): boolean {
-    return Boolean(this.stdin.isTTY);
-  }
+  // protected get isTerminal(): boolean {
+  //   return Boolean(this.stdin.isTTY);
```

Comment:
Do you remember what this is for? I think this could be deleted 🤔 .

Reply:
Oh, then this function should also be renamed, too.

May be good to use this function again where applicable.